docs/src/development/faq.md

```diff
@@ -79,7 +79,7 @@ Dedicated projects can deploy production to any public AWS or Azure region.
 
 'sh' is the short version of shell.
 
-According to Wikipedia™, in computing, a [shell](https://en.wikipedia.org/wiki/Shell_(computing\)) is a user interface for access to an operating system's services. Generally, operating system shells use either a [command-line interface ](https://en.wikipedia.org/wiki/Command-line_interface) (CLI) or [graphical user interface](https://en.wikipedia.org/wiki/Graphical_user_interface) (GUI).  This is exactly what Platform.sh is about: Giving developers tools to build, test, deploy, and run great websites!
+According to Wikipedia™, in computing, a [shell](https://en.wikipedia.org/wiki/Shell_(computing)) is a user interface for access to an operating system's services. Generally, operating system shells use either a [command-line interface ](https://en.wikipedia.org/wiki/Command-line_interface) (CLI) or [graphical user interface](https://en.wikipedia.org/wiki/Graphical_user_interface) (GUI).  This is exactly what Platform.sh is about: Giving developers tools to build, test, deploy, and run great websites!
```

Comment:
The \) was needed to keep it from mis-rendering the closing ), which is supposed to be part of the link.  Does Hugo handle that differently than GitBook?

Reply:
Yeah, it doesn't show [correctly anyway](https://docs.platform.sh/development/faq.html#why-did-you-choose-the-sh-extension-for-your-domain) and removes the ) which breaks the link